packages/design-system/src/components/input/useNumericInput.tsx

```diff
@@ -52,7 +55,7 @@ export const useNumericInput = ({
    * Call external `onChange`
    */
   const handleBlur = useCallback(
-    (ev) => {
+    (ev: unknown) => {
```

Comment:
Why unknown, shouldn't it be an event?

Reply:
It is invoked both from a native DOM event listener with a DOM KeyboardEvent (through `useKeyDownEffect`), and through a React event listener (through `onBlur`). So it's either a DOM event or a React event, but we don't actually care, we just pass it on to the `onChange` function (which probably doesn't care either).

It could be typed as `KeyboardEvent | FocusEvent`, which the latter being imported from React, but I found that a bit confusing.

It could also be solved by having a wrapper listener for the React event that invoked it the other with `evt.nativeEvent` ensure it would always receive a proper DOM event, but still, I didn't see a purpose, because we don't actually consume the event.